nixos/modules/services/web-apps/onlyoffice.nix

```diff
@@ -222,6 +228,13 @@ in
             cp -r ${cfg.package}/etc/onlyoffice/documentserver/* /run/onlyoffice/config/
             chmod u+w /run/onlyoffice/config/default.json
 
+            #Allow LAN-range IPs if set true
+            ${lib.optionalString (cfg.allowLocalConnections) ''
+            sed -i 's/"allowPrivateIPAddress": false/"allowPrivateIPAddress": true/' /run/onlyoffice/config/default.json
+            sed -i 's/"allowMetaIPAddress": false/"allowMetaIPAddress": true/' /run/onlyoffice/config/default.json
+            ''}
+
+
             # Allow members of the onlyoffice group to serve files under /var/lib/onlyoffice/documentserver/App_Data
             chmod g+x /var/lib/onlyoffice/documentserver
 
```

Comment:
I imagine we want to do the following:

- add a freeform json type option
- use pkgs.writeJSON on it
- merge it into the default.json in line 263

Can you do that? Thanks!

Reply:
I do not think I have the knowledge to do a free-form JSON type but I did replace my commits with a new one that just uses the boolean and jq as you requested